core/src/main/kotlin/com/bumble/appyx/navmodel/backstack/transitionhandler/BackStackFader.kt

```diff
@@ -8,16 +9,17 @@ import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.composed
 import androidx.compose.ui.draw.alpha
-import com.bumble.appyx.navmodel.backstack.BackStack
 import com.bumble.appyx.core.navigation.transition.ModifierTransitionHandler
 import com.bumble.appyx.core.navigation.transition.TransitionDescriptor
 import com.bumble.appyx.core.navigation.transition.TransitionSpec
+import com.bumble.appyx.navmodel.backstack.BackStack
 
 @Suppress("TransitionPropertiesLabel")
 class BackStackFader<T>(
     private val transitionSpec: TransitionSpec<BackStack.TransitionState, Float> = { spring() }
 ) : ModifierTransitionHandler<T, BackStack.TransitionState>() {
 
+    @SuppressLint("ModifierFactoryExtensionFunction")
```

Comment:
Maybe we should change API instead of suppressing it?
Also makes sense to regenerate lint.xml

Reply:
yes, but not in the scope of this issue. I'll update lint-baseline